### ISSUE TITLE
Replacing exif-js

### ISSUE THREAD

**Issue by MikeKovarik** [kartoteket/vue-image-upload-resize#36]
Hello,

I noticed you're having troubles with the dependecy exif-js (#15) which contains bugs but is no longer maintained (for over two years now).

I'd like to offer you a help with that. I'm a developer of another library [exifr](https://github.com/MikeKovarik/exifr). It's small, fast, configurable (*so just a few core files can be used to extract the photo's orientation instead of using the whole library*), supports more formats (tiff, heic), has fixes for bugs known to be in exif-js, etc...

I'd be happy to do a Pull Reqest swapping exif-js for exifr if you're open to that.

Thank you

**Comment by jsd219** [comment]
@MikeKovarik did he ever allow you to do a pull request?

**Comment by jsd219** [comment]
All of my images uploaded through my iPhone are turned and I am assuming it is do to the exif

**Comment by MikeKovarik** [comment]
@jsd219 no. Otherwise this issue would be closed. The problem you're referring to is a new behavior of safari on iOS (and soon to be in chrome as well). https://github.com/MikeKovarik/exifr/issues/25 fortunately exifr can work with it. Unfortunately this package doesn't use exifr

**Comment by MikeKovarik** [comment]
Also this https://mutiny.cz/exifr/examples/orientation.html

But it's broken on chrome 81 and newer because they just implemented the same behavior since I released last version. It will be fixed soon

**Comment by jsd219** [comment]
@MikeKovarik Any chance you can fork off of this and implement it yourself? This is a great plugin but unfortunately it appears the developer is not very active on it.

**Comment by MikeKovarik** [comment]
@jsd219 unfortunately not. I would be happy to do a PR to this repo if the author was active and willing to accept it. But I do not want to fork and maintain this repo. I don't want to step on anyone's toes, nor would I have enough time to maintain it.

**Comment by svale** [comment]
Sorry, unfortunately I've not had time to actively maintain the repository lately. 

I initially was a bit hesitant  about switching to another exif-lib, but that was probably an overly defensiv and passiv judgement call on my side. 

Please @MikeKovarik , if you still feel up to it, I would be very happy to accept a PR that implements your lib.  We'll merge it into a new major version release to safeguard against any unforeseen breaking changes.

Thank you for the patience!

(cc @jsd219 )

**Comment by bbugh** [comment]
I created a PR #49 to hopefully help move this forward. We are having trouble with #41 and I was hoping this would resolve it, but it does not appear to.

@svale and @MikeKovarik I wonder if you could take a look?
